src/libraries/System.Text.Json/src/System/Text/Json/Writer/JsonWriterOptions.cs

```diff
@@ -43,6 +44,23 @@ public bool Indented
             }
         }
 
+        /// <summary>
+        /// Defines the text used as indent by the <see cref="Utf8JsonWriter"/> when <see cref="Indented"/> is enabled
+        /// By default, the JSON is written with 2 white spaces.
+        /// </summary>
+        public string IndentText
+        {
+            get => _indentText ?? JsonConstants.DefaultIndent;
+            set
+            {
+                if (value is null)
+                {
+                    ThrowHelper.ThrowArgumentNullException(nameof(value));
+                }
+                _indentText = value;
```

Comment:
Shouldn't there be a minimal amount of validation, e.g. checking that the string is all whitespace characters?